start-preview.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+docker build -t kafka-site-preview .
+
+docker run -dit --rm --name mypreview -p 8080:80 -v "$PWD":/usr/local/apache2/htdocs/ kafka-site-preview
+
+echo "You can stop the preview server by running `docker stop mypreview`"
```

Comment:
What about doing something like this to make it even easier to use?

```suggestion
#!/usr/bin/env bash

set -euo pipefail

clean() {
  docker stop site-preview
  exit 0
}

trap clean SIGINT

docker build -t kafka-site-preview .

docker run -dit --rm --name site-preview -p 8080:80 -v "$PWD":/usr/local/apache2/htdocs/ kafka-site-preview

echo "The preview is available on http://localhost:8080."
echo "Press CTRL+C to stop it."

read -r -d '' _ </dev/tty

```

Reply:
Hi, thanks for the suggestion, I am not a shell expert so I might be missing something here, I think we achieve similar effect by not running the container in detached mode.

Then ctrl + c will work, do you think that's acceptable?

Reply:
Also, can you explain what `read -r -d '' _ </dev/tty` is trying to achieve?

Reply:
If you know a better way to achieve the same, yes do it